app/views/directives/notifications/notification-body.html

```diff
@@ -0,0 +1,61 @@
+<div
+  uib-dropdown
+  class="dropdown pull-right dropdown-kebab-pf"
+  ng-if="notification.actions && notification.actions.length > 0">
```

Comment:
I may just eliminate this.  We aren't actually adding any actions, its a leftover from the example template for the drawer.